File: 20221017-MaxAndMin/app/src/main/java/MaxAndMin.java
public class MaxAndMin {
    public String solution(String s) {
      String[] numbers = s.split(" ");

      int newMaxNumber = Integer.parseInt(numbers[0]);;
      int maxNumber = Integer.parseInt(numbers[0]);
      for(String element : numbers) {
        int firstNumber = Integer.parseInt(numbers[0]);
        maxNumber = Math.max(maxNumber,Integer.parseInt(element));
        newMaxNumber = Math.max(maxNumber,newMaxNumber);
      }

      int newMinNumber = Integer.parseInt(numbers[0]);
      int minNumber = Integer.parseInt(numbers[0]);

      for(String element : numbers) {
        int firstNumber = Integer.parseInt(numbers[0]);
        minNumber = Math.min(minNumber,Integer.parseInt(element));
        newMinNumber = Math.min(minNumber,newMinNumber);
      }
      String answer ="";

      answer = newMinNumber + " " + newMaxNumber;

      return answer;
    }
}

File: 20221018-MaxAndMin-Js/MaxAndMin.js
// 배열 생성
// 배열안들의 원소를 어떻게 비교해줄까?
export function getMax(convertedToNumberArray) {
  const max = convertedToNumberArray[convertedToNumberArray.length - 1];
  return max;
}

export function getMin(convertedToNumberArray) {
  const min = convertedToNumberArray[0];
  return min;
}

export function solution(s) {
  const stringNumberArray = s.split(' ');

  const convertedToNumberArray = stringNumberArray.map((element) => Number(element));
  convertedToNumberArray.sort((a, b) => a - b);

  const max = getMax(convertedToNumberArray);

  const min = getMin(convertedToNumberArray);

  const answer = `${min} ${max}`;

  return answer;
}
